halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/security/SamlValidator.java

```diff
@@ -30,10 +31,13 @@
 import org.apache.commons.lang3.StringUtils;
 import org.apache.http.client.methods.HttpGet;
 import org.apache.http.impl.client.HttpClientBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class SamlValidator extends Validator<Saml> {
+  @Autowired SecretSessionManager secretSessionManager;
```

Comment:
This is already inherited from `Validator`, you don't need to define it here.

Reply:
Ah, thanks! Updated.

Reply:
Ah, thanks! Updated.